pkg/acs/calacs/acscte/getcteflag.c

```diff
@@ -91,9 +92,21 @@ static int checkPCTE (Hdr *phdr, ACSInfo *acs, int *missing, int *nsteps) {
             return (status);
         }
 
-        if (GetTabRef (acs->refnames, phdr,
-                       "PCTETAB", &acs->pcte, &acs->pctecorr))
-            return (status);
+        if (acs->pcteTabNameFromCmd && *acs->pcteTabNameFromCmd != '\0')
+        {
+            char msgBuffer[256];
```

Comment:
Instead of magic number, I suggest you use one of the pre-defined constants. For example, `ACS_LINE` from `acs.h` should do well enough for this purpose.